core/src/main/resources/hudson/model/ComputerSet/index.jelly

```diff
@@ -56,8 +56,8 @@ THE SOFTWARE.
 
     <j:set var="monitors" value="${it._monitors}"/>
     <j:set var="tableWidth" value="${3}"/>
-
-    <table id="computers" class="jenkins-table sortable">
+    <t:setIconSize/>
```

Comment:
I've not ran this locally but why do we need `t:setIconSize` twice on this page?